MobileBuy/buy/build.gradle

```diff
@@ -128,12 +128,6 @@ dependencies {
     androidTestCompile 'com.google.dexmaker:dexmaker:1.1'
     androidTestCompile 'com.google.dexmaker:dexmaker-mockito:1.1'
 
-
-    testCompile 'org.mockito:mockito-core:1.9.5'
```

Comment:
This is "testCompile" instead of "androidTestCompile", so it's not needed anymore since we got rid of JUnit tests.